src/styles.ts

```diff
@@ -117,7 +120,7 @@ export const styles = css`
 	.line::before {
 		content: '';
 		position: absolute;
-		display: block;
+		display: var(--line-display, block);
```

Comment:
The line is already [exported as a part](https://github.com/Neovici/cosmoz-input/blob/master/src/render.ts#L35), so if you are trying to hide the line, you can do:

```css
cosmoz-input::part(line) {
    display: none;
}

Reply:
I am trying to hide the line, but only on the color input. Should it be something like:
`cosmoz-input[type="color"]::part(line){
display: none;
}`

?

Reply:
But this type of styling should be part of the component itself, because it is important to hide the line by default. So you would not use that selector, which is for styling the component from the outside. Instead you should include code in comsoz-input that hides the line by default, when the type is color. Something like:

```css
:host([type=color]) .line {
  display: none;
}

Reply:
You marked this comment as resolved, but you did not revert the change, nor explain why the change is necessary.

Reply:
My mistake, I did add the `display: none` property whenever the host is of type="color", but I haven't deleted the attribute in the `.line` class.

I did already squash the commits, so the description got lost. I'll specify that in the next commit message.